deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java

```diff
@@ -16,13 +16,14 @@
 
 package bnymellon.codekatas.deckofcards.list.immutable;
 
+import java.util.Collections;
+import java.util.List;
 import java.util.Random;
 
 import bnymellon.codekatas.deckofcards.Rank;
 import bnymellon.codekatas.deckofcards.Suit;
 import com.google.common.base.Joiner;
-import com.google.common.collect.HashMultiset;
-import com.google.common.collect.Multiset;
+import com.google.common.collect.*;
```

Comment:
Please specify individual classes in imports.

Reply:
Done!